src/v2/view-builder/views/ov/ChallengeOktaVerifyFastPassView.js

```diff
@@ -59,10 +64,15 @@ const Body = BaseForm.extend(Object.assign(
       const checkPort = () => {
         return request({
           url: getAuthenticatorUrl('probe'),
-          // in loopback server, SSL handshake sometimes takes more than 100ms and thus needs additional timeout
-          // however, increasing timeout is a temporary solution since user will need to wait much longer in worst case
-          // TODO: OKTA-278573 Android timeout is temporarily set to 3000ms and needs optimization post-Beta
-          timeout: BrowserFeatures.isAndroid() ? 3000 : 100
+          /*
+          OKTA-278573 in loopback server, SSL handshake sometimes takes more than 100ms and thus needs additional
+          timeout however, increasing timeout is a temporary solution since user will need to wait much longer in
+          worst case.
+          TODO: Android timeout is temporarily set to 3000ms and needs optimization post-Beta.
+          OKTA-365427 introduces probeTimeoutMillis; but we should also consider probeTimeoutMillisHTTPS for
+          customizing timeouts in the more costly Android and other (keyless) HTTPS scenarios.
+          */
+          timeout: BrowserFeatures.isAndroid() ? 3000 : probeTimeoutMillis
```

Comment:
I think I'm missing something, but where is this _configured_? For what I can see, the Android timeout and default `probeTimeoutMillis` are hardcoded in these source files.

Reply:
`probeTimeoutMillis` is configured on the server-side (CCS property) and retrieved by SIW via `/idp/idx/introspect`. Only if not found is current default of 100ms is used. Note that currently this enhancement will not affect Android which has a probe timeout of 3000ms (due to keyless HTTPS).